File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/SalesOrderSourceModel.ts
import { SalesOrderModel } from "./salesOrderModel";

export interface SalesOrderSourceModel {
    salesOrderSourceId: number;
    salesOrderSourceCode: string;
    salesOrderSourceNameEn: string;
    salesOrderSourceNameAr: string;
    color: string;
    icon: string;
    displayOrder: number | null;
    isActive: boolean | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;
 
}

File: SofiForce.Models/Models/EntityModels/SalesOrderSourceModel.cs
using System;

namespace Models
{
    public class SalesOrderSourceModel
    {
        public int SalesOrderSourceId { get; set; }
        public string SalesOrderSourceCode { get; set; }
        public string SalesOrderSourceNameEn { get; set; }
        public string SalesOrderSourceNameAr { get; set; }
        public string Color { get; set; }
        public string Icon { get; set; }
        public int? DisplayOrder { get; set; }
        public bool? IsActive { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}
